blueprints/ember-weak-ref/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  normalizeEntityName: function() {
+    // this prevents an error when the entityName is
+    // not specified (since that doesn't actually matter
+    // to us
+  },
+
+  afterInstall: function() {
+    return this.addAddonToProject('ember-weakmap', '^0.4.0');
```

Comment:
I'm guessing because my previous PR moving it to a dep wasn't versioned which is why it didn't work on my end consuming it in another addon?